docs/guided_tour_summary_pages_requests.md

```diff
@@ -0,0 +1,136 @@
+
+We'll use the ROUND function to clean up some of this data and make it more readable. The query below converts the bytes to KB by dividing by 1024.  And then rounds all the numbers.
+
+```
+SELECT pages.url pageUrl, 
+       count(*) reqImg, 
+       ROUND(SUM(respBodySize)/1024) imgKB, 
+       ROUND(MAX(respBodySize)/1024) largestImgKB,
+       ROUND(MAX(respBodySize) / SUM(respBodySize),2) largestImgKBPercent
+FROM `httparchive.summary_pages.2018_09_01_desktop`  pages
+INNER JOIN `httparchive.summary_requests.2018_09_01_desktop`  requests
+ON pages.pageid = requests.pageid
+WHERE requests.type = "image"
+GROUP BY pageUrl
+ORDER BY imgKB DESC
+```
+
+![example results](./images/guided_tour_summary_requests_pages_join_example4.jpg)
+
+The results above show us the number of images, the image weight, the size of the largest image, and the percentage of image weight represented by that largest image. In the example we explored earlier - http://thaliahallchicago.com/ had 80 images, with the largest image being 4,452KB.   This image represents 4% of the image weight, indicating that there are most likely multiple large images on the page.
+
+Since there are 1.3 million pages in the HTTP Archive, we have 1.3 million rows in this resultset. Let's aggregate the results from this query now to learn something about the data.
+
+```
+SELECT largestImgKBPercent, 
+       ROUND(imgKB/512)*512 imageweightbin, 
+       count(*) pages
+FROM (
+    SELECT pages.url pageUrl, 
+           count(*) reqImg, 
+           ROUND(SUM(respBodySize)/1024) imgKB, 
+           ROUND(MAX(respBodySize)/1024) largestImgKB,
+           ROUND(MAX(respBodySize) / SUM(respBodySize),2) largestImgKBPercent
+    FROM `httparchive.summary_pages.2018_09_01_desktop`  pages
+    INNER JOIN `httparchive.summary_requests.2018_09_01_desktop`  requests
+    ON pages.pageid = requests.pageid
+    WHERE requests.type = "image"
+    GROUP BY pageUrl
+    ORDER BY imgKB DESC
+) 
+GROUP BY largestImgKBPercent, imageweightbin
+```
+![example results](./images/guided_tour_summary_requests_pages_join_example5.jpg)
+
+This resultset has 6863 rows, which is easier to work with. Let's export it to Google Sheets and then see what we can learn from it. 
+
+In Google Sheets, we can creat a pivot table to cross tabulate the results. For example we can look at the Largest Image % vs the Image Weight Bin by using `largestImgKBPercent` as the rows, `imageweightbin` as the columns, and `pages` as the values. Now instead of 6863 rows, we have 100 rows and a large number of colums. We can also limit the number of columns using a filter, and in this example I've configured a filter on imageweightbin so that we're looking at pages with less than 5MB of images (this accounts for 90% of sites in the HTTPArchive that load images).

```

Comment:
```suggestion
In Google Sheets, we can create a pivot table to cross tabulate the results. For example we can look at the Largest Image % vs the Image Weight Bin by using `largestImgKBPercent` as the rows, `imageweightbin` as the columns, and `pages` as the values. Now instead of 6863 rows, we have 100 rows and a large number of columns. We can also limit the number of columns using a filter, and in this example I've configured a filter on imageweightbin so that we're looking at pages with less than 5MB of images (this accounts for 90% of sites in the HTTP Archive that load images).
```